CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Change Log for MIGraphX
+
+Full documentation for MIGraphX is available at [MIGraphX Github Pages](https://rocmsoftwareplatform.github.io/AMDMIGraphX/doc/html/).
+
+## MIGraphX 2.5 for ROCm 5.5.0
+
+### Added
+- Y-Model feature to store tuning information with the optimized model
+- Accuracy checker tool based on ONNX Runtime
+- ONNX Operators parse_split, and Trilu 
+- Build support for ROCm MLIR
+- Added migraphx-driver flag to print optimizations in python (--python)
+
+
+### Optimizations
+- Improved performance of Transformer based models
+- Improved performance of the Pad, Concat, Gather, and Pointwise operators
+- Improved onnx/pb file loading speed
+
+
+### Fixed
+- Improved parsing Tensorflow Protobuf files 
+- Resolved various accuracy issues with some onnx models
+- Resolved a gcc-12 issue with mivisionx
+- Improved support for larger sized models and batches
+- Use --offload-arch instead of --cuda-gpu-arch for the HIP compiler
```

Comment:
- Changes inside JIT to use `float` accumulator for large reduce ops of `half` type to avoid overflow.
- Changes inside JIT to temporarily use cosine to compute sine function.

Reply:
Done